cupy/core/raw.pyx

```diff
@@ -258,14 +263,18 @@ cdef class RawModule:
             the template kernel ``func1<T>`` and non-template kernel ``func2``.
             Strings in this tuple must then be passed, one at a time, to
             :meth:`get_function` to retrieve the corresponding kernel.
+        log_stream (str): Whether the output of nvcc should be displayed in
+            'stdout' or stored to a 'file'. For example, `log_stream('file.txt')`
+            will write to output to file.txt. This variable is not used by 
+            nvrtc.
 
     .. note::
         Each kernel in ``RawModule`` possesses independent function attributes.
     """
     def __init__(self, *, str code=None, str path=None, tuple options=(),
                  str backend='nvrtc', bint translate_cucomplex=False,
                  bint enable_cooperative_groups=False,
-                 name_expressions=None):
+                 name_expressions=None, str log_stream=None):
```

Comment:
backend='nvrtc' and log_stream != None are not compatible, we should add a check

Reply:
I think Matt will try to support both 😁